src/contexts/Maci.tsx

```diff
@@ -27,11 +27,40 @@ import {
   type MaciContextType,
   type MaciProviderProps,
 } from "./types";
+import type { Ballot, Vote } from "~/features/ballot/types";
 
 export const MaciContext = createContext<MaciContextType | undefined>(
   undefined,
 );
 
+export const sumBallot = (votes?: Vote[]) =>
+(votes ?? []).reduce(
+  (sum, x) => sum + (!isNaN(Number(x?.amount)) ? Number(x.amount) : 0),
+  0,
+);
+
+export const ballotContains = (id: string, ballot?: Ballot) => {
+  return ballot?.votes.find((v) => v.projectId === id);
```

Comment:
`return ballot?.votes?.find((v) => v.projectId === id);`

add `?` after `votes`, else if I don't have ballots, there will be an error
<img width="989" alt="image" src="https://github.com/privacy-scaling-explorations/maci-rpgf/assets/9023842/464f4920-c19c-4221-a53c-607002bd2786">